rcsb/utils/seq/UniProtUtils.py

```diff
@@ -531,14 +543,15 @@ def doGeneLookup(self, geneName, taxId, reviewed=False):
         """ """
         rL = []
         try:
-            baseUrl = self.__urlPrimary
-            endPoint = "uniprot"
+            # baseUrl = self.__urlPrimary  # Comment back in once all other methods are updated with new UniProt API
+            baseUrl = "https://rest.uniprot.org"  # new UniProt API baseUrl
+            endPoint = "uniprotkb/search"
```

Comment:
@piehld the `doLookup` method points to legacy UniProt whereas `doGeneLookup` points to the new API. Am I understanding this correctly?

Reply:
Yes, this is just temporary to correct the current set of Azure and ETL workflow issues. The reason why I didn't update the `doLookup()` method yet is because the way the new API works is so different from how it used to work (See "Python example" API documentation for ID mapping in middle of page here: https://www.uniprot.org/help/id_mapping).

Obviously we will need to update everything to work with the new API, but for now I decided to just keep using the legacy API until we have the time to make all those changes.

Reply:
Ok thank you.